packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -373,63 +380,84 @@ function TokensPanel({
     }
   }
 
+  function onSearchInputChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setErrorMessage('')
+    setNewToken(event.target.value)
+  }
```

Comment:
Can we memo this, so we don't pass a new reference every time to `SearchPanelTable`?